slave/process-o365-contacts-export/short_desc

```diff
@@ -0,0 +1 @@
+Package for perun service - export of emails for o365 purpose
```

Comment:
Please replace with following:
"Package for perun service - export Perun managed o365 emails"